io/plugins/eu.esdihumboldt.hale.io.xsd/src/eu/esdihumboldt/hale/io/xsd/reader/XmlSchemaReader.java

```diff
@@ -962,6 +965,49 @@ else if (object instanceof XmlSchemaParticle) {
 			}
 			// </sequence>
 		}
+		else if (particle instanceof XmlSchemaAll) {
+			// <all>
+			XmlSchemaAll all = (XmlSchemaAll) particle;
+
+			// create group only if necessary (all groups that appear exactly
+			// once will result in no group if not forced)
```

Comment:
Actually the group should be eliminated automatically. Not sure where that functionality is hooked up though.